tools/PI/DevHome.PI/BarWindowHorizontal.xaml.cs

```diff
@@ -141,21 +146,21 @@ private void SetDefaultPosition()
         _monitorRect = GetMonitorRectForWindow(_viewModel.ApplicationHwnd ?? ThisHwnd);
         var screenWidth = _monitorRect.right - _monitorRect.left;
         this.Move(
-            (int)(((screenWidth - Width) / 2) * _dpiScale) + _monitorRect.left,
-            (int)(_WindowPositionOffsetY * _dpiScale));
+            (int)((screenWidth - (Width * _dpiScale)) / 2) + _monitorRect.left,
+            (int)_WindowPositionOffsetY);
 
         // Get the saved settings for the ExpandedView size. On first run, this will be
         // the default 0,0, so we'll set the size proportional to the monitor size.
         // Subsequently, it will be whatever size the user sets.
         var settingSize = Settings.Default.ExpandedLargeSize;
         if (settingSize.Width == 0)
         {
-            settingSize.Width = _monitorRect.Width * 2 / 3;
+            settingSize.Width = (int)((_monitorRect.Width * 2) / (3 * _dpiScale));
```

Comment:
Will throw a `Divide By Zero` error if `_dpiScale` is `DEVICE_SCALE_FACTOR_INVALID`

Unless this value is checked before this code lights up.

Reply:
according to the documentation it should be set to a valid value on error: https://learn.microsoft.com/en-us/windows/win32/api/shellscalingapi/nf-shellscalingapi-getscalefactorformonitor
"If the function call fails, this value points to a valid scale factor so that apps can opt to continue on with incorrectly sized resources."